WMF Framework/WMFExploreFeedContentController.m

```diff
@@ -9,26 +9,50 @@
 #import "WMFAssertions.h"
 #import <WMF/WMF-Swift.h>
 
+NSString * const WMFExploreFeedContentControllerBusyStateDidChange = @"WMFExploreFeedContentControllerBusyStateDidChange";
+
 static const NSTimeInterval WMFFeedRefreshTimeoutInterval = 12;
 static NSTimeInterval WMFFeedRefreshBackgroundTimeout = 30;
+static const NSString *kvo_WMFExploreFeedContentController_operationQueue_operationCount = nil;
```

Comment:
super minor, but I used to do 

static const NSString *kvo_WMFExploreFeedContentController_operationQueue_operationCount = "kvo_WMFExploreFeedContentController_operationQueue_operationCount";

just in case I wanted to print out the context in the debugger. In practice I think it helped maybe once ¯\_(ツ)_/¯